app/BUILD.bazel

```diff
@@ -488,6 +500,7 @@ android_library(
     visibility = ["//visibility:private"],
 )
 
+# Final app library
```

Comment:
What does 'final' mean?

Reply:
I just meant that it was the 'final' library before building the binary - like the main library for the whole app. I can remove 'final', it is confusing.

Reply:
I think we should just be descriptive about what this represents, e.g.:

``Library containing all activity, fragment, and view-based UI flows in the app.`` or similar.

Reply:
Done